routes/tasks.js

```diff
@@ -2,13 +2,13 @@ const express = require("express");
 const router = express.Router();
 const authenticate = require("../middlewares/authenticate");
 const tasks = require("../controllers/tasks");
-const { createTask, updateTask, updateSelfTask } = require("../middlewares/validators/tasks");
+const { createTask, updateTask, updateSelfTask, getTasks } = require("../middlewares/validators/tasks");
 const authorizeRoles = require("../middlewares/authorizeRoles");
 const { APPOWNER, SUPERUSER } = require("../constants/roles");
 const assignTask = require("../middlewares/assignTask");
 const cache = require("../utils/cache");
 
-router.get("/", cache(), tasks.fetchTasks);
+router.get("/", getTasks, cache(), tasks.fetchTasks);
```

Comment:
can we change the validator name to something else, here i am thinking its getting task

Reply:
Sure

Reply:
Done